### PR TITLE
Gh 432 stop erroneous error message about configuration environment variable firing during unit tests

### PR DESCRIPTION
**Pull request by developer01189998819991197253**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,4 +36,8 @@ node_modules
 .settings
 .settings/
 .classpath
-.classpath/
\ No newline at end of file
+.classpath/
+
+#added files auto generated by vscode
+.vscode/*
+library/shade-hadoop-client/dependency-reduced-pom.xml
```
Comment (m09526) [inline .gitignore]:
```suggestion
dependency-reduced-pom.xml
```

File: pom.xml [hunk 2]
```diff
@@ -299,6 +300,7 @@
                         <exclude>${exclude.tests}</exclude>
                     </excludes>
                     <systemPropertyVariables>
+                        <PALISADE_REST_CONFIG_PATH>example/example-model/src/main/resources/configRest.json</PALISADE_REST_CONFIG_PATH>
```
Comment (m09526) [inline pom.xml]: Could this be an environment variable instead of a System property?

File: pom.xml [hunk 3]
```diff
@@ -331,6 +333,7 @@
                         <include>**/*IntegrationTests.java</include>
                     </includes>
                     <systemPropertyVariables>
+                        <PALISADE_REST_CONFIG_PATH>example/example-model/src/main/resources/configRest.json</PALISADE_REST_CONFIG_PATH>
```
Comment (m09526) [inline pom.xml]: As above.

File: service/config-service/src/main/java/uk/gov/gchq/palisade/config/service/ConfigUtils.java [hunk 4]
```diff
@@ -48,6 +48,8 @@ private ConfigUtils() {
     public static String retrieveConfigurationPath() {
         String path = System.getenv(ConfigConsts.CONFIG_SERVICE_PATH);
         if (isNull(path)) {
+            path = System.getProperty(ConfigConsts.CONFIG_SERVICE_PATH);
+        } else if (isNull(path)) {
```
Comment (m09526) [inline service/config-service/src/main/java/uk/gov/gchq/palisade/config/service/ConfigUtils.java]: See above comments, if it is possible to set environment variables in failsafe and Surefire, then these two lines can be removed.

File: pom.xml (hunk 3 above)
Comment (developer01189998819991197253) [reply pom.xml]: made change and committed to branch

File: pom.xml (hunk 2 above)
Comment (developer01189998819991197253) [reply pom.xml]: made change and committed to branch

File: service/config-service/src/main/java/uk/gov/gchq/palisade/config/service/ConfigUtils.java (hunk 4 above)
Comment (developer01189998819991197253) [reply service/config-service/src/main/java/uk/gov/gchq/palisade/config/service/ConfigUtils.java]: made change and committed to branch

**Pull request merged**